src/react-styleguide/src/components/Buttons/Button.js

```diff
@@ -40,33 +47,54 @@ const Button = ({href, type='default', size='large', outline=false, children}) =
 
 Button.propTypes = {
   href: React.PropTypes.string,
-  type: React.PropTypes.oneOf(['success', 'warning', 'danger', 'primary', 'default']),
-  size: React.PropTypes.oneOf(['large', 'small', 'cta']),
-  outline: React.PropTypes.bool,
+  type: React.PropTypes.oneOf(types),
+  size: React.PropTypes.oneOf(sizes),
+  options: React.PropTypes.arrayOf(React.PropTypes.oneOf(options)),
   children: React.PropTypes.string.isRequired
 }
 
+const buttonExampleList = [
+  {
+    children: 'Default'
+  },
+  {
+    type: 'primary',
+    size: 'cta',
+    children: 'Primary'
+  },
+  {
+    type: 'success',
+    options: ['outline'],
+    children: 'Success'
+  },
+  {
+    type: 'warning',
+    size: 'small',
+    children: 'Warning'
+  },
+  {
+    type: 'danger',
+    size: 'small',
+    options: ['outline'],
+    children: 'Danger'
+  }
+]
+
 /**
 * Button Examples for Style Guide
 */
 export const ButtonExamples = () => {
   return (
     <div className='flex flex-row justify-center flex-wrap items-center'>
-      <div className='mb2'>
-        <Button>Default</Button>
-      </div>
-      <div className='mb2 mh2'>
-        <Button type='primary' size='cta'>Primary</Button>
-      </div>
-      <div className='mb2 mh2'>
-        <Button type='success' outline>Success</Button>
-      </div>
-      <div className='mb2 mh2'>
-        <Button type='warning' size='small'>Warning</Button>
-      </div>
-      <div className='mb2 mh2'>
-        <Button type='danger' size='small' outline>Danger</Button>
-      </div>
+      {buttonExampleList.map((btn, i) => {
+        return (
+          <div className='mb2 mh2' key={i}>
+            <Button options={btn.options} {...btn}>
```

Comment:
remove `options={btn.options}` since `outline: true` will get passed with the other options with `{...btn}`